source/includes/reference/_borderless.md

```diff
@@ -5,74 +5,140 @@
+    "modificationTime": "2018-10-20T18:37:21.116Z",
+    "active": true,
+    "eligible": true,
+    "balances": [
+      {
+        "currency": "GBP",
+        "amount": {
+          "value": 20.61,
+          "currency": "GBP"
+        },
+        "reservedAmount": {
+          "value": 16.2,
+          "currency": "GBP"
+        },
+        "bankDetails": [
+          {
+            "id": 1000038,
+            "profileId": <your profile id>,
+            "currency": "GBP",
+            "accountHolderName": "Oliver Wilson",
+            "bankCode": {
+              "sortCode": "23-14-70"
+            },
+            "accountNumber": {
+              "default": "11010101"
+            },
+            "address": {
+              "default": {
+                "firstLine": "TransferWise",
+                "secondLine": "56 Shoreditch High Street",
+                "postCode": "E1 6JJ",
+                "city": "London",
+                "country": "United Kingdom",
+                "stateCode": null
+              }
+            },
+            "limits": {
+              "daily": {
+                "currency": "GBP",
+                "value": null
+              },
+              "yearly": {
                 "currency": "GBP",
-                "amount": {
-                    "value": 10999859,
-                    "currency": "GBP"
-                },
-                "reservedAmount": {
-                    "value": 0,
-                    "currency": "GBP"
-                },
-                "bankDetails": null
+                "value": null
+              }
             },
-            {
-                "balanceType": "AVAILABLE",
+            "translations": {
+              "accountLimits.daily": "24 hour receive limit",
+              "accountNumber.default": "Account number",
+              "address.default": "Address",
+              "bankCode.sortCode": "UK Sort Code",
+              "accountLimits.yearly": "Annual receive limit",
+              "accountHolderName": "Account Holder"
+            },
+            "status": "OPEN"
+          }
+        ]
+      },
+      {
+        "currency": "EUR",
+        "amount": {
+          "value": 0,
+          "currency": "EUR"
+        },
+        "reservedAmount": {
+          "value": 0,
+          "currency": "EUR"
+        },
+        "bankDetails": [
+          {
+            "id": 8,
+            "profileId": <your profile id>,
+            "currency": "EUR",
+            "accountHolderName": "Oliver Wilson",
+            "bankCode": {
+              "swiftCode": "DEKTDE7GXXX"
+            },
+            "accountNumber": {
+              "iban": "DE10 7002 1110 6050 0000 00"
+            },
+            "address": {
+              "default": {
+                "firstLine": "Handelsbank",
+                "secondLine": "Elsenheimer Str. 41",
+                "postCode": "80687",
+                "city": "München",
+                "country": "Germany",
+                "stateCode": null
```

Comment:
I find the system a bit inconsistent in addresses. We use country name in full but stateCode not state name. Just observation.